src/vs/platform/windows/electron-main/windows.ts

```diff
@@ -151,10 +151,7 @@ export function defaultBrowserWindowOptions(accessor: ServicesAccessor, windowSt
 			// Enable experimental css highlight api https://chromestatus.com/feature/5436441440026624
 			// Refs https://github.com/microsoft/vscode/issues/140098
 			enableBlinkFeatures: 'HighlightAPI',
-			sandbox: true,
-			// TODO(deepak1556): Should be removed once migration is complete
-			// https://github.com/microsoft/vscode/issues/239228
-			enableDeprecatedPaste: true,
```

Comment:
Lets keep this setting, we still have usages from webview that needs to be removed https://github.com/microsoft/vscode/blob/e6b351d8f10e4d94627ffda790c1fba5857f22d8/src/vs/workbench/contrib/webview/browser/webviewElement.ts#L721